### PR TITLE
feat: basic app setup and create layouts with design

### PR DESCRIPTION
**Pull request by lkononenko**
- created layoutMini for all pages before customer is logged in the wallet as it has a different design from pages after the login
- added color theme switcher, favicon

### CODE REVIEW

File: packages/apps/dev-wallet/src/App/layout-mini.css.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { atoms, tokens, vars } from '@kadena/react-ui/styles';
+import { style } from '@vanilla-extract/css';
+import { globalStyle } from '@vanilla-extract/css';
+
+globalStyle('body', {
```
Comment (ferreroltd) [inline packages/apps/dev-wallet/src/App/layout-mini.css.ts:5]: I would move this to a globals overwrite or something

Comment (lkononenko) [reply packages/apps/dev-wallet/src/App/layout-mini.css.ts]: sounds better indeed, just moved it

File: packages/apps/dev-wallet/src/App/layout.tsx [hunk 2]
```diff
@@ -31,14 +42,26 @@ export const Layout: FC = () => {
         }
       >
         <NavHeaderLinkList>
-          <Link to="/">
-            <Text bold>DX-Wallet</Text>
-          </Link>
-          <Link to="/networks">
-            <Text bold>Network</Text>
-          </Link>
+          <NavHeaderLink href="/">
```
Comment (ferreroltd) [inline packages/apps/dev-wallet/src/App/layout.tsx:45]: we need to use react-router links instead

**Review by ferreroltd — APPROVED** [review]
left some remarks but looks good!

File: packages/apps/dev-wallet/src/App/layout.tsx (hunk 2 above)
Comment (lkononenko) [reply packages/apps/dev-wallet/src/App/layout.tsx]: updated!

File: packages/apps/dev-wallet/src/App/routes.tsx [hunk 3]
```diff
@@ -41,33 +42,37 @@ const ProtectedRoute: FC<
...
+            path="/backup-recovery-phrase/:keySourceId/write-down"
+            element={<WriteDownRecoveryPhrase />}
+          />
+          <Route
+            path="/account-discovery/:keySourceId"
+            element={<AccountDiscovery />}
+          />
+          ,
+          <Route path="/accounts/:account" element={<p>Account</p>} />,
+          <Route path="/sig-builder" element={<SignatureBuilder />} />,
```
Comment (javadkh2) [inline packages/apps/dev-wallet/src/App/routes.tsx:67]: I think these two routes should go to the normal layout

Comment (lkononenko) [reply packages/apps/dev-wallet/src/App/routes.tsx]: updated

**Review by javadkh2 — APPROVED** [review]

**Pull request merged**